src/deepxtrace/diagnose.py

```diff
@@ -22,6 +22,11 @@
 import uuid
 import os
 import torch
+try:
```

Comment:
Does this code have to be after `import torch`? Adds some comments would be better. Thx.